### PR TITLE
updated mobile styling draft

### LINKED ISSUE #30: Fix Mobile Styling
# Fix Mobile Styling
Fix the styling for the mobile view of the website so images fit properly and seamlessly on the new design.

## Acceptance Criteria
- [x] Adjust graphics sizes and positions for mobile view
- [x] Reformat text for mobile view

### PR DESCRIPTION
**Pull request by silverdiamant**

### CODE REVIEW

File: static/css/main.css [hunk 1]
```diff
@@ -14,12 +14,12 @@ body {
   margin: 0 auto;
 }
 
-@media only screen and (max-width: 993px) {
+@media only screen and (min-width: 1500px) {
```
Comment (atoneyd) [inline static/css/main.css]: What's the reason you had to change these numbers? did the previous boundaries not work?

File: static/css/main.css [hunk 2]
```diff
@@ -81,6 +82,7 @@ a {
 .banner-info .apply {
   margin-top: 40px;
   display: none;
+  text-align: right;
```
Comment (atoneyd) [inline static/css/main.css:85]: I think maybe we should just keep everything centered on mobile, including the HackNC text, countdown, and banner graphic. The graphics could also be a bit bigger and centered at the top

File: static/css/main.css [hunk 3]
```diff
@@ -514,6 +591,13 @@ nav ul a:hover::before {
   }
 }
 
+/* [Mobile] FAQ */
+@media only screen and (max-width: 736px) {
+  .faq-header {
+    font-size: 1.7rem;
+  }
```
Comment (atoneyd) [inline static/css/main.css:655]: The text for the actual questions could be a bit smaller

**Review by atoneyd — CHANGES_REQUESTED** [review]
Looks pretty good! besides the things I commented, I think we just need to 
1. Resize the wave transition graphics between section (right now they still look zoomed in, except for the ones before the sponsor section and footer) 
2. The "About" section looks squished; we should have the text above the crow graphic. like how it is on fall2022.hacknc.com or pearlhacks.com

File: static/css/main.css (hunk 1 above)
Comment (silverdiamant) [reply static/css/main.css]: I was just testing the width I think but forgot to change it back so I fixed it!

**Comment by silverdiamant** [comment]
I didn't entirely zoom out on the wave transitions for the header section because it doesn't leave enough space for the header and rest of the code but if looks better zoomed out I can decrease the header text sizes.

**Review by atoneyd — APPROVED** [review]
Looks good enough for now, in the future we can discuss any additional design changes we think of.

**Pull request merged**